stdlib/2/unittest.pyi

```diff
@@ -153,11 +152,22 @@ class TestSuite(Testable):
     def debug(self) -> None: ...
     def countTestCases(self) -> int: ...
 
-# TODO TestLoader
-# TODO defaultTestLoader
+class TestLoader:
+    def loadTestsFromTestCase(self,
+                              testCaseClass: Union[TestCase, None]) -> TestSuite: ...
```

Comment:
The py3 stubs say `Type[TestCase]`, which I think is accurate. Also, from the code it doesn't look like None is actually accepted.